WWDC/VideoPlayerViewController.swift

```diff
@@ -122,6 +122,12 @@ final class VideoPlayerViewController: NSViewController {
         }.disposed(by: disposeBag)
     }
 
+    override func viewDidAppear() {
+        super.viewDidAppear()
+
+        view.window?.makeFirstResponder(playerView)
```

Comment:
This conflicts with `SessionsTableViewController.viewDidAppear` such that changing tabs doesn't result in player becoming first responder. Actually, the touch bar controls appear then disappear.

I think the table view wants first responder so you can move around the list with the keyboard... possibly? But, the explicit `makeFirstResponder` call can be removed from `SessionsTableViewController` because in `AppCooridinator` I did this:

```
scheduleItem.initialFirstResponder = scheduleController.listViewController.tableView
```

Which I believe made the call unnecessary and allows you to keep this hear.

Reply:
Really? On my tests, the player view successfully became the first responder, activating the Touch Bar. It has the side effect of stopping arrow navigation on the sidebar when you select the currently playing item tho, I'm not sure which one is worse (breaking arrow navigation or not having the Touch Bar controls show up automatically) :|

Reply:
Hey, just got the chance to double check. Switching tabs definitely causes the controls to flash on then back off as the table view takes first responder back.

Regardless of that, I like where you're heading. I agree that it makes sense that the controls be visible anytime a video is playing unless there are plans to have other touch bar controls for different app interactions.

Reply:
> unless there are plans to have other touch bar controls for different app interactions

Touch Bars can be combined, so that's not an issue :)